src/mocks/handlers.ts

```diff
@@ -995,4 +996,83 @@ export const handlers = [
       { status: 200 }
     );
   }),
+  http.get('/images', ({ request }) => {
+    const url = new URL(request.url);
+    const imageParams = url.searchParams;
+    const primary = imageParams.get('primary');
+    const entityId = imageParams.get('entity_id');
+
+    const thumbnail2 =
```

Comment:
I would rather image data like this lives in the json file to avoid having large byte strings in it. Same as in the upload above.